packages/jest-expo/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### 🎉 New features
 
+- [android][web][expo-asset] Added methods `bytes`, `arrayBuffer`, `content` and `contentArrayBuffer` to read contents of `Asset`s. ([#41567](https://github.com/expo/expo/pull/41567) by [@psnet](https://github.com/psnet))
```

Comment:
imo there's no need to copy this into a changelog since it doesn't relate to jest-expo, also we're rebuild those mocks for sdk release